server/controllers/admin.controller.js

```diff
@@ -15,7 +15,7 @@ exports.getAllAdmin = async (req, res) => {
     try {
         const response = await adminService.findAll(queryParams);
         if (response) {
-            responseUtil.successResponse(res, MessageUtil.success, response);
+            responseUtil.successResponse(res, MessageUtil.success, response[0]);
```

Comment:
Adding total count of rows is good, in such cases if no. of key:values of response always remains same , then better to switch to object.

Reply:
Since we are using aggregate pipelines, Mongo returns a array of objects, where the data of our interest is within the first element. I have made changes on services to fetch the first element of the array, thereby sending object back to the controller.